src/document/reducer.ts

```diff
@@ -314,29 +384,35 @@ export function baseReducer<T, A extends Action, L>(
         operationResultingStateParser,
     } = options;
 
-    const _action = { ...action };
-    const skipValue = skip || 0;
+    let _action = { ...action };
+    let skipValue = skip || 0;
     let newDocument: Document<T, A, L> = { ...document };
-    // let clipboard = [...document.clipboard];
+    let reuseLastOperationIndex = false;
 
     const shouldProcessSkipOperation =
         !ignoreSkipOperations &&
         (skipValue > 0 || ('index' in _action && _action.skip > 0));
 
     // ignore undo redo for now
```

Comment:
I guess this can be removed now :)

Reply:
fixed: [e66bcca](https://github.com/powerhouse-inc/document-model/pull/89/commits/e66bcca058fbef5dbb2543fc155ac580437e1a61)